scripts/postinstall.js

```diff
@@ -3,8 +3,8 @@ var os = require('os');
 
 switch (os.type()) {
   case "Windows_NT":
-    exec("mkdir build\\contracts\\ && copy node_modules\\witnet-ethereum-bridge\\build\\contracts\\*.json build\\contracts")
+    exec("mkdir build\\contracts\\ && copy node_modules\\witnet-ethereum-bridge\\build\\**\\**\\*.json build\\contracts")
```

Comment:
Does `**\\**` work on windows?

Reply:
@guidiaz will confirm :rofl:

Reply:
My assumption is that this path will not be expanded by Windows but rather by `mkdir` itself :shrug: